ncl/ncl_entries/specx_specxy_anal.ipynb

```diff
@@ -0,0 +1,2260 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# specx_specxy_anal\n",
+    "\n",
+    "This example demonstrates how to perform 1 and 2-dimensional analysis using the `scipy.signal` module. To learn more about the functions and keyword arguments available in this package, read Scipy's [Signal Processing documentation](https://docs.scipy.org/doc/scipy/reference/signal.html).\n",
```

Comment:
Is this really 2D or should the description be something more like cross-spectral analysis?